.travis.yml

```diff
@@ -10,6 +10,7 @@ addons:
 
 script:
   - ./gradlew check
+  - chmod +x start_local_a+_env.sh && ./start_local_a+_env.sh
```

Comment:
Just a note: perhaps all these steps should be joined with &&s so that if one step fails, subsequent steps are not run at all. But as there haven't be such functionality before, there is no need to change it in this PR.

Reply:
why not? :)